core/src/key/mod.rs

```diff
@@ -15,19 +13,21 @@
 /// crate::key::node::all                /${nd}
 /// crate::key::node::lq                 /${nd}!lq{lq}{ns}{db}
 ///
+/// crate::key::root::access::all        /&{ac}
+/// crate::key::root::access::gr         /&{ac}!gr{gr}
```

Comment:
Do we reserve `&` for accesses or should this be `/&ac{ac}`?

Reply:
My understanding from our conversation is that we do reserve that separator for access methods, yeah.

Reply:
Okay 👍